build/lib/helm_require.sh

```diff
@@ -71,11 +73,11 @@ do
   else
     USE_TAG=$TAG
   fi
-  # If IMAGE_TAG is different from HELM_TAG, we are using images from upstream.
+  # If HELM_USE_UPSTREAM_IMAGE is true, we are using images from upstream.
   # Though we pull images directly from upstream for build tooling checks (i.e.
   # get images shasums), we will use cached images in the helm charts. Cached
   # images follow the convention of ${PROJECT_NAME}/${UPSTREAM_IMAGE_NAME}.
-  if [ "${IMAGE_TAG}" != "${HELM_TAG}" ]
+  if [ "${HELM_USE_UPSTREAM_IMAGE}" == true ]
```

Comment:
I needed to separate "use upstream images" from "helm tag is different from image tag" because the image GIT_TAG value is different from the HELM_GIT_TAG value we use to build the helm charts.

I realize we probably don't love this, so I'm very open to alternative suggestions.

Reply:
Thanks for clarifying 👍  Yeah, we have different IMAGE_TAG and HELM_TAG values as well as different GIT_TAG and HELM_GIT_TAG values unfortunately.